airbase/airtable.py

```diff
@@ -232,46 +250,39 @@ async def get_base(
                 if bases:
                     base = bases[0]
             if base:
-                self.logger.info(f"Fetched Base with {key if key else 'value'}: {value}")  # noqa: E501
+                self.logger.info(
+                    f"Fetched Base with {key if key else 'value'}: '{value}'"
+                )  # noqa: E501
                 return base
         else:
-            error_msg = self.get_error_message(
-                method="get",
-                obj='base',
-            )
-            self.raise_or_log_error(error_msg)
+            error_msg = f"ERROR: Base with {key if key else 'value'}:'{value}' not found"  # noqa: E501
```

Comment:
is this prefix needed? I assume the error logger handles that?

Reply:
Corrext. Prefix is not needed.